mobile.js

```diff
@@ -32,4 +110,81 @@ document.documentElement.addEventListener('click', (e) => {
     hamburguerIcon.classList.remove('hide');
     closeIcon.classList.remove('show');
   }
-});
\ No newline at end of file
+  if (e.target.matches('.see-project')) {
+    const nboton = e.srcElement.getAttribute('data-id');
+    createmodal(nboton);
+    const modalWindow = document.querySelector('.modal-container');
+    modalWindow.style.display = 'block';
+  }
+  if (e.target.matches('.close-xbtn')) {
+    const modalWindow = document.querySelector('.modal-container');
+    modalWindow.style.display = 'none';
+  }
+  if (e.target.matches('.ic_menu *')) {
+    hamburguerIcon.classList.toggle('hide');
+    closeIcon.classList.toggle('show');
+    menuIcon.classList.toggle('hide');
+  }
+});
+
+// -------------------------------------------------create projects
```

Comment:
[OPTIONAL] Avoid inline comments in the middle of the code. They will make your project look messy. If you need to review a previous version of your code, you can always use git history. 🔖